.github/workflows/documentation.yml

```diff
@@ -0,0 +1,22 @@
+name: terraform-docs-gh-actions
+on:
+  - pull_request
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: .
+    steps:
+    - name: checkout project
+      uses: actions/checkout@v2
+      with:
+        ref: automate-docs
+    - name: Render terraform docs inside the USAGE.md and push changes back to branch
+      uses: terraform-docs/gh-actions@v0.6.1
+      with:
+        working-dir: .
+        output-file: USAGE.md
+        output-method: inject
+        git-push: "true"
+        #template: #Usage<br><br><!--- BEGIN\_TF\_DOCS ---><br><!--- END\_TF\_DOCS ---><br>
```

Comment:
I'm not sure, but this or something like it might be needed if we use README.md to tell terraform-docs the section of the README that can be generated.